front/app/components/SignUpIn/AuthProviders.tsx

```diff
@@ -60,17 +61,18 @@ export type AuthProvider = 'email' | SSOProvider;
 
 const AuthProviders = memo<Props & InjectedIntlProps>(
   ({
-    metaData,
+    azureAdLoginEnabled,
     className,
-    onAuthProviderSelected,
-    goToOtherFlow,
-    tenant,
-    passwordLoginEnabled,
-    googleLoginEnabled,
     facebookLoginEnabled,
-    azureAdLoginEnabled,
     franceconnectLoginEnabled,
+    goToOtherFlow,
+    googleLoginEnabled,
     intl: { formatMessage },
+    metaData,
+    onAuthProviderSelected,
+    passwordLoginEnabled,
+    tenant,
+    viennaLoginEnabled,
```

Comment:
This is the new entry, the rest of the diff is coming from me sorting the items alphabetically :abcd:.

Reply:
I wouldn't spend time sorting this stuff alphabetically. We're not doing this anywhere (I also don't see why we would need to do this), so that will be a huge job and time not well spent.

Reply:
It's really not a big thing and just one short-cut in my editor, I wouldn't worry to much about such a minor thing.

I think it also provides value, because a sorted list is easier to do mental gymnastics with in general. Looking at the appConfiguration, [the currencies](https://github.com/citizenlabdotco/citizenlab/blob/CL2-6937_omniauth_vienna_saml/front/app/services/appConfiguration.ts#L315-L495) for example are alphabetically sorted.

Reply:
I think our brains work differently 😁